dotnet/src/Connectors/Connectors.Memory.CosmosDB/CosmosDBMemoryRecord.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.Memory;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Connectors.Memory.CosmosDB;
```

Comment:
Any reason not `Microsoft.SemanticKernel.Connectors.Memory.CosmosDB` like the others?

Reply:
updating.